### PR TITLE
Allow views to be overridden with vendor content

### PR DESCRIPTION
**Pull request by bear454**
... and set up the Plan view to be friendly for this.

### CODE REVIEW

**Comment by bear454** [comment]
CC @stefanotorresi

**Review by stefanotorresi — CHANGES_REQUESTED** [review]

File: app/views/plans/show.html.haml [hunk 1]
```diff
@@ -3,9 +3,8 @@
   %div.card-body
     %p= markdown(t('page_subtitle.plan_info'))
     %hr
-    = loading_icon(hide: @show_output.present?)
-    %pre.pre-scrollable
-      %code.output= @show_output
+    = loading_icon(hide: @plan.present?)
+    = render('plans/plan', plan: @plan) if @plan.present?
```
[the remark this answers is not in the document]
Comment (bear454) [reply app/views/plans/show.html.haml]: Resolved in 4f64e56 .

File: spec/features/plan_spec.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@
     end
 
     it 'loads without a pre-generated plan' do
-      expect(find('code.output')).to have_no_content
+      expect(page).not_to have_selector('code.output')
```
[the remark this answers is not in the document]
Comment (bear454) [reply spec/features/plan_spec.rb]: Resolved in 4f64e56.

**Review by stefanotorresi — APPROVED** [review]
perfect, @bear454! :+1: Thanks a lot :beers:

**Pull request merged**